src/projectservice/index.js

```diff
@@ -70,9 +72,63 @@ app.post('/createProject', async (req, res) => {
   })
 })
 
+// Endpoint for deleting a project
+app.delete('/deleteProject/:projectID', async (req, res) => {
+  let token = req.body.token
+  if (!token) {
+    const { username, password } = req.body
+    token = await getToken(username, password)
+  }
+  const projectID = req.params.projectID
+  try {
+    const status = await deleteProject(projectID, token)
+    const data = {
+      // eslint-disable-next-line quote-props
+      'status': status.message,
+      // eslint-disable-next-line quote-props
+      'TaigaAPIResponseStatus': status.success
+    }
+    if (status.success) {
+      res.status(201).send(data)
+    } else {
+      res.status(500).send({
+        message: 'Project ID is invalid'
+      })
+    }
+  } catch (error) {
+    res.status(500).json({ error: error.message })
+  }
+})
+
+// Endpoint for Updating a Project
+app.patch('/updateProject/:projectID', async (req, res) => {
```

Comment:
Again, frontend is not storing any ID at their end, we will have to modify this to accept projectName and fetch projectId based on that.